hal/targets/hal/TARGET_STM/TARGET_STM32L4/sleep.c

```diff
@@ -46,13 +46,27 @@ void deepsleep(void)
 {
     // Stop HAL systick
     HAL_SuspendTick();
-
-    // Request to enter STOP mode with regulator in low power mode
-    HAL_PWR_EnterSTOPMode(PWR_LOWPOWERREGULATOR_ON, PWR_STOPENTRY_WFI);
+  
+    if(__HAL_RCC_PWR_IS_CLK_ENABLED())
```

Comment:
please update to follow code style guidelines https://docs.mbed.com/docs/getting-started-mbed-os/en/latest/Full_Guide/Code_Style/